model/metrics/gen-ai.yaml

```
@@ -0,0 +1,39 @@
+groups:
+  - id: metric_attributes.gen_ai
+    type: attribute_group
+    brief: 'This group describes GenAI metrics attributes'
+    attributes:
+      - ref: error.type
+        requirement_level:
+          conditionally_required: "if the operation ended in an error"
+      - ref: server.address
+        requirement_level: required
+      - ref: server.port
+        requirement_level: required
+      - ref: gen_ai.response.model
+        requirement_level: recommended
+      - ref: gen_ai.request.model
+        requirement_level: recommended
```

Tis one required on spans and we should require it on metrics. (we can be proven wrong later one, but let's be consistent now)
```suggestion
        requirement_level: required
```